### DISCUSSION TITLE
能不能加入下载MC地图

### DISCUSSION THREAD

**Discussion by lihaoran0313** [Meloong-Git/PCL#134] [提问] [answered]
PCL 能不能像HMCL一样加入下载MC地图功能

**Comment by WTP016-CN** [comment]
国内暂未有合理且统一的地图下载途径
国外的 MinecraftMaps 使用 MediaFire 需要梯子
另外新功能提议请使用 Issues 而不是 Discussion

**Comment by miner233** [comment]
CurseForge 里面也有地图啊

**Comment by Pigeon0v0** [comment, marked as answer]
与 #45 重复
